mindsdb/integrations/handlers/twelve_labs_handler/README.md

```diff
@@ -71,4 +87,18 @@ WHERE query = 'search query';
 
 Here, the `query` column is the name of the column containing the search queries as specified in the `query_column` parameter of the `CREATE MODEL` statement.
 
-Note: At the moment, only a single query can be specified in the `WHERE` clause of the query. The `JOIN` clause for making multiple predictions will be added in a future release.
\ No newline at end of file
+Note: At the moment, only a single query can be specified in the `WHERE` clause of the query. The `JOIN` clause for making multiple predictions will be added in a future release.
+
+### Summarization
+```sql
+SELECT *
+FROM mindsdb.twelve_labs_summarization
+WHERE video_id = 'video_1';
+```
+
+Here, the video IDs that were indexed by a model can be found by running a `DESCRIBE` statement on the it. The URL or file path of the video will be available in the `video_reference` column. The following is an example of how to run such a `DESCRIBE` statement,
+```sql
+DESCRIBE mindsdb.twelve_labs_summarization.indexed_videos;
+```
+
```

Comment:
Let's include a response of the DESCRIBE as an example here

Reply:
Done.